### PR TITLE
brand: update twitter card metadata

### LINKED ISSUE #539: provide tags/metadata for easier sharing on Social sites
Sharing links pointing to flatpak.org should be nice and branded. See https://github.com/flatpak/flatpak.github.io/issues/537

### PR DESCRIPTION
**Pull request by jimmac**
Fixes https://github.com/flatpak/flatpak.github.io/issues/539

### CODE REVIEW

File: data/site.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: Flatpak
-domain: http://flatpak.org
+domain: https://flatpak.org
+#domain: http://localhost:4567
```
Comment (mwleeds) [inline data/site.yml]: Is this intentional?

**Review by jimmac — COMMENTED** [review]
removing the localhost comment

**Comment by AsciiWolf** [comment]
> removing the localhost comment

I still see the localhost comment in site.yml.

**Comment by AsciiWolf** [comment]
Should this PR still be merged?

**Comment by AsciiWolf** [comment]
I have created a [separate PR](https://github.com/flatpak/flatpak.github.io/pull/625) for the http -> https url change.

**Comment by AsciiWolf** [comment]
@jimmac Should this PR still be merged?

**Comment by AsciiWolf** [comment]
I have merged the following PR that seems to collide with this one.

https://github.com/flatpak/flatpak.github.io/pull/635

@jimmac Could you update this PR? Thanks!

**Comment by jimmac** [comment]
I'm afraid my git-fu is too weak to get through 40 commit conflict resolution.

**Pull request closed without merging**